addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -9,6 +9,7 @@
 		<supported-bridge-type-refs>
 			<bridge-type-ref id="envisalink" />
 			<bridge-type-ref id="it100" />
+            <bridge-type-ref id="tcpserver" />
```

Comment:
There is now some mix of tabs and spaces in this file - could you apply the code formatter on it so that it is homogenous?